docs/cookbook/persistence/sqlite.md

```diff
@@ -0,0 +1,46 @@
+# SQLite Database
+
+The SQLite persistence is a great default for many use cases, and is a good place to start if you're not sure which persistence to choose from.
+
+By default when using the `bdk_wallet` library, all information about the wallet is held in memory, and will be destroyed upon termination of the process unless saved to persistence. 
+
+When information important to the wallet is added to it, the wallet will add it to its staged area. Whenever you want to save this information to persistence, call the `Wallet.take_staged()-> Option<ChangeSet>` method; if the method returns a changeset, you can then persist it to your SQLite database using `db.write(changeset)`.
+
+The operations that affect the wallet and produce a changeset are things like:
+
+- Revealing new addresses
+- Sync operations that pick up new UTXOs
+
+Once those things are persisted, upon loading of the database changeset the wallet would be able to rehydrate its [TxGraph](), which includes UTXOs, transaction history, and latest blocks known to the wallet. This means that a wallet that's been loaded from such a persitence will not require a _Full Scan_ but rather simply a _Sync_.
+
+See our page on the [difference between the full scan and sync operations](../syncing/full-scan-vs-sync.md) for more on this topic.
+
+## Example
+
+The sqlite wallet does not require any additional dependencies above the `bdk_wallet` dependency:
+
+```toml
+[dependencies]
+bdk_wallet = { version = "=1.0.0-beta.1" }
+```
```

Comment:
But it does need the `rusqlite` feature:

```suggestion
The sqlite wallet does not require any additional dependencies above the `bdk_wallet` dependency:

```toml
[dependencies]
bdk_wallet = { version = "=1.0.0-beta.1", features = ["rusqlite"] }
```
```